caravel/assets/javascripts/SqlLab/components/QuerySearch.jsx

```diff
@@ -34,13 +36,41 @@ class QuerySearch extends React.Component {
     const val = (db) ? db.value : null;
     this.setState({ databaseId: val });
   }
-  insertParams(baseUrl, params) {
-    return baseUrl + '?' + params.join('&');
+  getTimeFromSelection(selection) {
+    switch (selection) {
```

Comment:
function should take the attr, not to be evaluated.
@mistercrunch @ascott  - what is your take here?